docs/ai/conceptual/agents.md

```diff
@@ -0,0 +1,86 @@
+
+For example, you can use instructions to tell the agent that it is helping people manage emails, and to explain its decisions as it makes them.
+
+### Plugins
+
+You use [plugins](/semantic-kernel/agents/plugins/?tabs=Csharp) to do things an LLM can't do alone, such as:
+
+- Retrieving data from external data sources
+- Knowing what time it is
+- Performing complex math
+- Completing tasks in the real world
+- Memorizing and recalling information
+
+For example, an LLM can't send an email, so to add that function to a chat app, you'd need a plugin.
+
+Make sure you clearly document the functions in your plugins&mdash;planners use this information to determine what functions are available.
+
+### Planners
+
+A [planner](/semantic-kernel/agents/planners/?tabs=Csharp) can analyze available functions and come up with alternate ways to reach the goal.
+
+Calling plugin functions isn't always efficient. For example, say you want to sum the numbers between 1 and 100. You could call a math plugin, but the LLM would need to make a separate call for each number.
+
+Moreover, the best sequence and combination of functions to reach a goal depends on the details. For example, suppose you're building an email chat helper app, so you include a plugin to enable sending emails. However, some emails might need a different action, such as a meeting request without RSVP&mdash;sending a reply isn't necessary, but adding a calendar item is. A planner looks at all available functions and comes up with efficient ways to reach goals.
+
+## Copilots add user interaction
+
+Process automation has many benefits, but sometimes the user needs to make decisions along the way. An agent can't automate user actions. That's where copilots come in.
+
+An agent in your email chat app might produce the following plan for sending an email:
+
+1. Get the user's email address and name
+1. Get the email address of the recipient
+1. Get the topic of the email
+1. Generate the subject and body of the email
+1. Send the email
+
+Very handy, but what if the user doesn't like the email body? A copilot adds a user interaction step to the plan:
+
+1. Get the user's email address and name
+1. Get the email address of the recipient
+1. Get the topic of the email
+1. Generate the subject and body of the email
+**1. Review the email with the user and make adjustments**
+1. Send the email
+
+### Semantic Kernel Chat Copilot app
+
+To get started with copilots, try the [Semantic Kernel Chat Copilot](/semantic-kernel/chat-copilot/), a reference application for building a chat experience with an AI agent.
+
+## Related content
+
+- [Develop AI agents using Azure OpenAI and the Semantic Kernel SDK](/training/paths/develop-ai-agents-azure-open-ai-semantic-kernel-sdk/)
```

Comment:
I'd link to the OpenAI function calling doc once we have it, since that explains how we describe to OpenAI what functions it can call.